sencha-workspace/SlateDemonstrationsStudent/app/controller/Viewport.js

```diff
@@ -13,9 +13,9 @@ Ext.define('SlateDemonstrationsStudent.controller.Viewport', {
     views: [
         'CompetencyCard',
         'Dashboard',
-        'DashboardController',
         'RecentProgress',
-        'OverviewWindow'
+        'OverviewWindow',
+        'OverviewWindowController'
```

Comment:
@aliyeysides all the same  things I said above apply here, PLUS you should not be putting the same values in multiple controller's `views` config